python/cuml/test/test_fil.py

```diff
@@ -572,11 +588,11 @@ def test_lightgbm(tmp_path, num_classes, n_categorical):
                                  n_estimators=num_round)
         lgm.fit(X_fit, y)
         lgm.booster_.save_model(model_path)
+        lgm_preds = lgm.predict(X).astype(int)
```

Comment:
If possible, also use `X_fit` for LightGBM and `X` for FIL, to be consistent between training and prediction.